warehouse/warehouse/scripts/verify_published.py

```diff
@@ -91,6 +91,11 @@ def verify_dataset(src_path, base_path, scripts_path, out_path, pbar, **kwargs):
 
 
 def main():
+
+    # elided as unreliable - timing issue?
```

Comment:
I wonder if this is fix-able?  After a publication occurs, any process that immediately queries the state of the esgf server fails to reflect the publication - Sasha explained it has to  do with the time it takes to propagate the state to all servers.

Reply:
P.S.  I was surprised to discover that vi recognizes "FIXME" as a language token, and automatically highlights it.

Reply:
Not sure if it is fixable, but at least it indicates that it should be revisited in the future.

Yup, FIXME:, NOTE:, TODO:, and others can be picked up by text editors/IDEs since they are common practice in comments. I have a VSCode extension that automatically highlights these words and displays them as a list in one of the side windows.